integration_tests/grpc_rest/dclauth/helpers.go

```diff
@@ -519,4 +609,98 @@ func AuthDemo(suite *utils.TestSuite) {
 	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{firstModel}, accountName, testAccount)
 	require.Error(suite.T, err)
 	require.True(suite.T, sdkerrors.ErrUnknownAddress.Is(err))
+
+	// Ensure that new account is exist in revoked account
+	receivedRevokedAccount, err := GetRevokedAccount(suite, testAccAddr)
+	require.NoError(suite.T, err)
+	require.Equal(suite.T, testAccAddr, receivedRevokedAccount.GetAddress())
+	require.Equal(suite.T, []dclauthtypes.AccountRole{dclauthtypes.Vendor}, receivedRevokedAccount.GetRoles())
+
+	// Jack proposes for reading the revoked account
```

Comment:
Typo in `reading`. Should be `re-adding`

Reply:
Done! ✅